src/domain/relation/explore.ts

```diff
@@ -1,5 +1,5 @@
 
-import type Requester from '../authentication/Requester';
+import Requester from '../authentication/Requester';
```

Comment:
Why isn't this imported as a type anymore?

Reply:
Type restored